### ISSUE TITLE
inconsistent instrumentation of sticky toggle buttons

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/sun#460]
RectangularStickyToggleButton inherits from RectangularButtonView.  RoundButtonView has no a11y-specific options values, and uses the defaults in RectangularButtonView, which are:

```js
      // a11y
      tagName: 'button',
```

RoundStickToggleButton inherits from RoundButtonView.  In RoundButtonView:

```js
      tagName: 'input',
      inputType: 'button'
```

Why are these so different?  I'm guessing that RectangularStickyToggleButton RoundStickToggleButton should have similar a11y instrumentation, and the instrumentation should be handled in the base class where possible.

**Comment by jessegreenberg** [comment]
There is no reason for the difference, they are equivalent for a11y. I made it a `tagName: button`, though in #459 this will probably be removed entirely and added to the supertype RoundButtonView.
